packages/truffle-decoder/test/contracts/WireTest.sol

```diff
@@ -0,0 +1,143 @@
+pragma solidity ^0.5.10;
+pragma experimental ABIEncoderV2;
+
+contract WireTestParent {
+
+  event Done();
+
+  function inherited(uint[2] memory) public {
+    emit Done();
+  }
+
+  //no constructor
+}
+
+contract WireTest is WireTestParent {
+  constructor(bool status, bytes memory info, Ternary whoknows) public {
+    deepStruct["blornst"].length = 9;
```

Comment:
is blornst an adjective or a noun?

Reply:
It's a determiner, obviously.  "Look at blornst tree!"  "It's in blornst book I read."